pom.xml

```diff
@@ -72,7 +70,7 @@
     <dependency>
       <artifactId>jquery</artifactId>
       <groupId>org.webjars</groupId>
-      <version>3.3.1-2</version>
+      <version>3.7.1</version>
```

Comment:
It would be cleaner to put a versioning var in the `<properties></properties>` block and reference it here using `${...}` rather than inlining dependency verisons as has been done with `<thymeleaf-layout-dialect.version>...</thymeleaf-layout-dialect.version>` - same for the below.